cmake.spec

```diff
@@ -1,4 +1,4 @@
-### RPM external cmake 3.17.2
+### RPM external cmake 3.25.2
```

Comment:
Hi @forthommel is it needed to update cmake

Reply:
Hi @menglu21. I tried dropping CMake 3.20+ features in this gcc7-friendly CepGen version, let us try.